### ISSUE TITLE
XeGTAO and Fog Texture issues

### ISSUE THREAD

**Issue by Azyrion** [GarrettGunnell/AcerolaFX#14]
Hello I noticed that when I out the two shaders in the new optional path for Custome Shaders it has a conflict with other Shaders in textures.
![grafik](https://user-images.githubusercontent.com/29234550/192353196-2dd87a73-0149-4e8a-8934-d30b1c797c90.png)

**Comment by GarrettGunnell** [comment]
Yeah, this is because we declare a texture with the same name but GShade prioritizes the AO shader that it comes preinstalled with.

To fix this you can either delete those shaders that are causing the conflicts (`MC_SSAO.fx` and `PandaFX.fx`), check the `Do Not Load Unused Shaders` in the settings, or wait for my update coming out in the next few days that will resolve the texture conflict.
